frontend/src/pages/Mesh/target/TargetPanelEditor.tsx

```diff
@@ -1,36 +1,60 @@
 import * as React from 'react';
-import { dump } from 'js-yaml';
-import { aceOptions, yamlDumpOptions } from 'types/IstioConfigDetails';
-import { istioAceEditorStyle } from '../../../styles/AceEditorStyle';
 import AceEditor from 'react-ace';
 import ReactAce from 'react-ace/lib/ace';
 import { useKialiTheme } from '../../../utils/ThemeUtils';
 import { Theme } from '../../../types/Common';
 import { ConfigButtonsTargetPanel } from '../../../components/Mesh/ConfigButtonsTargetPanel';
+import { AceOptions } from 'react-ace/types';
+import { kialiStyle } from 'styles/StyleUtils';
+import { PFColors } from 'components/Pf/PfColors';
 
 interface TargetPanelEditorProps {
   configMap: string;
   targetName: string;
 }
 
+const aceOptions: AceOptions = {
+  highlightActiveLine: false,
+  highlightGutterLine: false,
+  maxLines: Infinity,
+  showLineNumbers: false,
+  showPrintMargin: false
+};
+
+const aceEditorStyle = kialiStyle({
+  marginTop: '0.5rem',
+  backgroundColor: PFColors.BackgroundColor100,
+  $nest: {
+    '& .ace_gutter': {
+      backgroundColor: PFColors.BackgroundColor100,
+      borderRight: 0
```

Comment:
```
,
      left: "-10px !important",
    },
```

What do you think about reducing a little bit the left margin?

Reply:
Agreed, I have moved the editor a bit to the left as suggested.